src/couch_epi_sup.erl

```diff
@@ -12,18 +12,33 @@
 
 -module(couch_epi_sup).
 
+%% ----------------------------------
+%% Important assumption
+%% ====================
+%% Keeper and codechange_monitor childspecs relie on underdocumented feature.
```

Comment:
And "underdocumented" I suppose - it's about having inadequate documentation while I believe documentation is fine, it's just about using undocumented behaviour.

Also header upper line need to be aligned by header width.